### PR TITLE
escape spaces in path on Windows OS

### PR DESCRIPTION
**Pull request by phannebohm**
- fixes ticket [#5971](https://trac.openmodelica.org/OpenModelica/ticket/5971)

...or at least I hope so. Can anyone with a Windows machine test this? Have a look at the ticket.

### CODE REVIEW

File: OpenModelica/StartTLMOpenModelica.bat [hunk 1]
```diff
@@ -10,7 +10,7 @@ REM 6 model-file
 
 rem *********************SET PATHS****************************************************
 rem set path to OpenModelica installation folder
-set OpenModelicaPath=%OPENMODELICAHOME%
+set "OpenModelicaPath=%OPENMODELICAHOME%"
```
Comment (adeas31) [inline OpenModelica/StartTLMOpenModelica.bat:13]: Think its not going to work. It should be `set OpenModelicaPath="%OPENMODELICAHOME%"` but i can test it tomorrow.

File: OpenModelica/StartTLMOpenModelica.bat [hunk 2]
```diff
@@ -10,7 +10,7 @@ REM 6 model-file
 
 rem *********************SET PATHS****************************************************
 rem set path to OpenModelica installation folder
-set OpenModelicaPath=%OPENMODELICAHOME%
+set "OpenModelicaPath=%OPENMODELICAHOME%"
 
 rem *********************SETUP SIMULATION*********************************************
 set OMC_Cmd=%OpenModelicaPath%/bin/omc.exe
```
Comment (adeas31) [inline OpenModelica/StartTLMOpenModelica.bat:16]: It doesn't work. Instead change this line to,

`set OMC_Cmd="%OpenModelicaPath%/bin/omc.exe"`

Comment (phannebohm) [reply OpenModelica/StartTLMOpenModelica.bat]: Thanks for reviewing. I changed it accordingly.

**Comment by adeas31** [comment]
Did you change the line endings of the file?

**Comment by phannebohm** [comment]
> Did you change the line endings of the file?

Yeah, my editor did that automatically, it's CRLF now (as usual for Windows files I believe). Should I undo that?

**Comment by adeas31** [comment]
In general CRLF is fine for a Windows bat file but maybe its better to just do the minimal change.

**Pull request merged**